Demo/Sources/Views/PlaybackSlider.swift

```diff
@@ -49,11 +49,13 @@ struct PlaybackSlider<ValueLabel>: View where ValueLabel: View {
         GeometryReader { geometry in
             ZStack(alignment: .leading) {
                 rectangle(opacity: 0.1)
+                    .background(.ultraThinMaterial)
                 rectangle(opacity: 0.3, width: geometry.size.width * CGFloat(progressTracker.buffer))
                 rectangle(width: geometry.size.width * CGFloat(progressTracker.progress))
             }
             .animation(.linear(duration: 0.5), value: progressTracker.buffer)
             .gesture(dragGesture(in: geometry))
+            .busy((progressTracker.player?.isBusy ?? false) && !progressTracker.isInteracting)
```

Comment:
I would suggest factoring out this code in an `isBusy` property instead of using a one-liner. This would also make it possible to check the `player` optional property with a `guard` statement.